test/spec/modules/prebidServerBidAdapter_spec.js

```diff
@@ -2692,6 +2692,26 @@ describe('S2S Adapter', function () {
       sinon.assert.match(actual.imp[0], sinon.match(ortb2Imp));
     });
 
+    it('setting adapterCode for default bidder', function () {
+      config.setConfig({ CONFIG });
+      adapter.callBids(REQUEST, BID_REQUESTS, addBidResponse, done, ajax);
+      server.requests[0].respond(200, {}, JSON.stringify(RESPONSE_OPENRTB));
+
+      const response = addBidResponse.firstCall.args[1];
+      expect(response).to.have.property('adapterCode', 'appnexus');
+    });
+
+    it('setting adapterCode for alternate bidder', function () {
+      config.setConfig({ CONFIG });
+      RESPONSE_OPENRTB.seatbid[0].bid[0].ext.prebid.meta.adaptercode = 'appnexus2'
+      adapter.callBids(REQUEST, BID_REQUESTS, addBidResponse, done, ajax);
+      server.requests[0].respond(200, {}, JSON.stringify(RESPONSE_OPENRTB));
+
+      const response = addBidResponse.firstCall.args[1];
+      expect(response).to.have.property('adapterCode', 'appnexus2');
+      delete RESPONSE_OPENRTB.seatbid[0].bid[0].ext.prebid.meta.adaptercode;
```

Comment:
minor point - it's better to deepClone the object, or put this in a finally block. Otherwise, if any future change causes this test to fail, the teardown logic will not run likely causing a cascade of confounding failures.

Reply:
Done